extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java

```diff
@@ -40,5 +42,48 @@
          */
         @ConfigItem(defaultValue = "true")
         public boolean enabled;
+
+        /**
+         * Build time configuration options for routes discovery.
+         */
+        @ConfigItem
+        public RoutesDiscoveryConfig routesDiscovery;
+    }
+
+    @ConfigGroup
+    public static class RoutesDiscoveryConfig {
+        /**
+         * Enable automatic discovery of routes during static initialization.
+         */
+        @ConfigItem(defaultValue = "true")
+        public boolean enabled;
```

Comment:
I assume this option is taken into account only if main is enabled. Therefore I'd expect to find it under `camel.main` rather than as a sibling of camel main.

Reply:
it is already, see https://github.com/apache/camel-quarkus/pull/400/files/e7a680d139f1e4d050cba26cb99572454976023f#diff-cd72e3ae157d3249a7dfe5705d1ef89bR50

Reply:
Thanks, and sorry for my misunderstanding.